libs/rpc/drivers/extension-content-driver.js

```diff
@@ -5,42 +5,73 @@
  */
 var ExtensionContentDriver = function () {
```

Comment:
I am not sure that we need this kind of queuing system. If I am understanding the code correctly, the driver will queue commands between switch of two (or more) tabs? If that is the case, I would simplify the approach and remove the queues. I think this scenario is so rare and it does not make sense to handle it.